### ISSUE TITLE
[pilight] Pilight 1.8 with OH2b1 failing connect correctly

### ISSUE THREAD

**Issue by GENETX** [openhab/openhab#3805]
Hi all,

I just gave OpenHAB 2b1 a test-drive. However, I seem to experience some problems with getting the old 1.x bindings to work out-of-the box. To be more specific: It seems that the openhab.cfg file does not get loaded correctly on startup. However, if I trigger a refresh by renaming this config file (e.g. to openhab1.cfg) then the 1.x bindings start to work. This is reproducable on each startup. Also, it does not matter if i create separate cg files for the bindings as suggested in the documentation ( https://github.com/openhab/openhab-distro/blob/master/docs/sources/migration.md ). I run OpenHAB 2b1, the online package, with the demo-package being installed on first start. I did not alter the original demo files, except that I did extend these. 

The rest of the config is as follows:
openhab.cfg
`pilight:livingroom.host=192.168.1.14
pilight:livingroom.port=5000
pilight:livingroom.delay=1000
`

demo.items
`...original demo items go here...
...
Switch	Light_Staande	"StaandeLamp"	<light>	(LIGHT,Lights)		{ pilight="livingroom#switch2" }
`
These snippets of configuration work flawlesly in OpenHAB 1.6 and when I refresh the config file as well. Neither bindings are working on startup, altough the log tells me that pilight is connected succesfully:
`14:21:22.587 [DEBUG] [g.dispatch.internal.ConfigDispatcher] - Processing config file 'pilight.cfg'   
20:52:26.667 [INFO ] [ng.pilight.internal.PilightConnector] - Established connection to pilight server at 192.168.1.14:5000                     
20:52:39.552 [INFO ] [ternal.PilightGenericBindingProvider] - pilight:livingroom item Light_Staande bound to device switch2               `
The first line is repeated upon refreshing the cfg. Perhaps this is a problem in the binding itself (wrong order of initialization perhaps?).

See also the discussion here:
https://community.openhab.org/t/oh1-config-not-loading/6249

**Comment by teichsta** [comment]
could you help with this one @idserda?

**Comment by idserda** [comment]
Is this still an issue with the latest OH2 build?

**Comment by GENETX** [comment]
I finally found some time to test OpenHAB2b2 and the updated pilight binding. The last snapshot from https://openhab.ci.cloudbees.com/job/openHAB1-Addons/lastSuccessfulBuild/artifact/bundles/binding/org.openhab.binding.pilight/target/ Build #1190 (22-mrt-2016 3:12:20) seems to work good :) . Unfortunately, it is not bundled in the OH2 binding lists so I had to install it manually using these commands:

```
bundle:install https://openhab.ci.cloudbees.com/job/openHAB1-Addons/lastSuccessfulBuild/artifact/bundles/binding/org.openhab.binding.pilight/target/org.openhab.binding.pilight-1.9.0-SNAPSHOT.jar
bundle:start org.openhab.binding.pilight
```

I created an pilight.cfg file manually just the way you'd expect it in OH2. Started it and worked. I expect that the mega PR from watou ( #3948 ) did the trick for this binding too.
